components/api/support.js

```diff
@@ -13,31 +14,34 @@ const Support = ({ contact_description, contact_link }) => {
 
         {contact_link ? (
           <>
-            {contact_link.includes("@") ? (
+            {contact_link.includes('@') ? (
```

Comment:
We must stop this kind of magic in the future. We must opt for a more declarative approach : use 2 fields `contact_email` & `contact_link`, the FD is then free to use one of them or the both.

Reply:
Noted